- minor adjustements for change notifications AEH delivery mode (#9623)
microsoftgraph/microsoft-graph-docs: concepts/change-notifications-delivery.md

Change notifications can be delivered in different ways to subscribers. If the main delivery mode for change notifications is through webhooks, it can be challenging to take advantage of webhooks for high throughput scenarios or when the receiver cannot expose a publicly available notificiation URL.  

> As the change notifications delivery mode is a platform-board capability, it is available for all resources that support Microsoft Graph change notifications.

Reviewer: I'm not sure what "platform-board" means. Will customers?

Suggested change:

Change notifications are available for all resources that support Microsoft Graph change notifications.

Author: ```suggestion
This change notifications delivery mode is available for all resources that support Microsoft Graph change notifications.
```

Using Azure Event Hubs to receive change notifications differs from webhooks in a few ways, including:

- You don't rely on publicly exposed notification URLs. The Event Hubs SDK will relay the notifications to your application.

> - You don't need to reply to the [notification URL validation](webhooks.md#notification-endpoint-validation). You will receive a validation message that you need to discard.

Reviewer: ```suggestion
- You don't need to reply to the [notification URL validation](webhooks.md#notification-endpoint-validation). You can ignore the validation message that you receive.
```

> **Note:** Create a separate policy for the application that listens to Event Hubs messages instead of reusing the same connection string you set in Azure KeyVault. This ensures that each component of the solution has only the permissions it needs and follows the least permissions security principle.

> > **Note:** Your application will receive validation messages whenever it creates a new subscription, you should discard this type of notifications. The example below represents the body of a validation message.

Reviewer: ```suggestion
> **Note:** Your application receives validation messages whenever it creates a new subscription, you should ignore these notifications. The example below represents the body of a validation message.
```
